src/lib/stitching/gravity/v2/stitching.ts

```diff
@@ -299,13 +299,14 @@ export const gravityStitchingEnvironment = (
           }
           `,
           resolve: async ({ image_url }, _args, context, info) => {
-            context.imageData = normalizeImageData(image_url)
-
             return info.mergeInfo.delegateToSchema({
               schema: localSchema,
               operation: "query",
               fieldName: "_do_not_use_image",
-              context,
+              context: {
+                ...context,
```

Comment:
ooo nice one

Reply:
Shallow copy here, as [noted ](https://artsyproduct.atlassian.net/browse/FX-4543?focusedCommentId=54867)in the ticket.